feat: add props zenModeEnabled and gridModeEnabled so host can control completely (#2901)
excalidraw/excalidraw: src/packages/excalidraw/CHANGELOG.md

### Features

> - Add `zenModeEnabled` and `gridModeEnabled` prop which enables zen mode and grid mode respectively [#2901](https://github.com/excalidraw/excalidraw/pull/2901). When this prop is used, the zen mode / grid mode will not show up in context menu is so it is fully controlled by host.

Reviewer: ```suggestion
- Add `zenModeEnabled` and `gridModeEnabled` prop which enables zen mode and grid mode respectively [#2901](https://github.com/excalidraw/excalidraw/pull/2901). When this prop is used, the zen mode / grid mode will be fully controlled by the host app.
```
